db/migrate/20170717140010_migrate_puppetdb_api_version_setting.rb

```diff
@@ -2,11 +2,11 @@
 
 class MigratePuppetdbApiVersionSetting < ActiveRecord::Migration[4.2]
   def up
-    puppetdb_address = Setting.where(category: 'Setting::Puppetdb', name: 'puppetdb_address').first.try(:value)
+    puppetdb_address = Setting.where(category: 'Setting', name: 'puppetdb_address').first.try(:value)
```

Comment:
It's not *needed*, was more for consistency.

Reply:
Technically I think you could be breaking the migration. If on a modern installation you set the version do 3 (which this migration does) you actually try to set an invalid version (since 165850e50476c474f80ee8f6ebf37d1967d63b10 dropped v1 and v3 support).

Reply:
Then https://github.com/theforeman/puppetdb_foreman/commit/165850e50476c474f80ee8f6ebf37d1967d63b10 should have dropped that migration? (I'll drop that hunk in this PR, but I still think the migration needs a change, or a rm)